pinecone/models/collections.go

```diff
@@ -11,11 +11,11 @@ import (
 
 // CollectionModel describes the collection data model.
 type CollectionModel struct {
-	Name      types.String `tfsdk:"name"`
-	Size      types.Int64  `tfsdk:"size"`
-	Status    types.String `tfsdk:"status"`
-	Dimension types.Int64  `tfsdk:"dimension"`
-	// VectorCount types.Int64  `tfsdk:"vector_count"`
```

Comment:
I added `VectorCount` back in generally and also swapped things to use `Int32` where it aligns with the OpenAPI spec.

Reply:
I think this is one of the fields we'd like to change to Int64. It's fine (and probably more correct) to use Int32 here for now as long as it doesn't cause a maintenance headache for when the `VectorCount` inevitably changes back to Int64 in the Protobuf definition.

Reply:
Since the underlying SDK is also using Int32 I'd prefer to leave it as an Int32 until we're actually ready to make the switch. Adjusting it in the codebase here to support the change isn't too bad, and we can tie it with the SDK changes.